Feature/minimal degree of autonomy (#726)
rl-institut/multi-vector-simulator: docs/MVS_Outputs.rst

        &OEM \epsilon \text{[0,1]}

.. _kpi_degree_of_autonomy:

Degree of autonomy (DA)
#######################

> The degree of autonomy describes the overall energy consumed minus the energy consumed from the grid divided by the overall energy consumed. Adapted from this definition `[2] <https://pvspeicher.htw-berlin.de/solarspeicherstudie/>`__.

Reviewer: ```suggestion
The degree of autonomy describes the overall energy consumed minus the energy consumed from the grid divided by the overall energy consumed. Adapted from this definition `[3] <https://pvspeicher.htw-berlin.de/solarspeicherstudie/>`__.
```
ahh just remembered: this should be [3] as it is different from the above reference 2 in OEM section

> The degree of autonomy describes the overall energy consumed minus the energy consumed from the grid divided by the overall energy consumed. Adapted from this (definition `[2] <https://pvspeicher.htw-berlin.de/solarspeicherstudie/>`__).

Reviewer 2: ```suggestion
The degree of autonomy describes the overall energy consumed minus the energy consumed from the grid divided by the overall energy consumed. Adapted from this definition `[2] <https://pvspeicher.htw-berlin.de/solarspeicherstudie/>`__.
```

> A DA close to zero shows high dependence on the grid operator, while a DA of 1 represents an autonomous system. Note that this key parameter indicator doesn't take into account the outflow from the system to the grid operator.

Reviewer 2: ```suggestion
A degree of autonomy close to zero shows high dependence on the grid operator, while a degree of autonomy of one represents an autonomous system. Note that this key parameter indicator does not take into account the outflow from the system to the grid operator (also called feedin).
```

As above, we apply a weighting based on Electricity Equivalent.

.. math::

>        DA <= \frac{\sum E_{demand,i} \cdot w_i - \sum E_{consumption,provider,j} \cdot w_j}{\sum E_{demand,i} \cdot w_i}

Reviewer 2: ```suggestion
       DA = \frac{\sum E_{demand,i} \cdot w_i - \sum E_{consumption,provider,j} \cdot w_j}{\sum E_{demand,i} \cdot w_i}
```
